Policies/Kubernetes/allowed-volume-types/azurepolicy.json

```diff
@@ -0,0 +1,63 @@
+{
+  "properties": {
+    "displayName": "Ensure only allowed Volume Types are used in Kubernetes Cluster",
+    "policyType": "Custom",
+    "mode": "Microsoft.Kubernetes.Data",
+    "description": "This policy ensures only allowed volume types are used inside a Kubernetes cluster. For instructions on using this policy, please visit https://aka.ms/kubepolicydoc",
+    "metadata": {
+      "version": "1.0.0-preview",
+      "category": "Kubernetes PSP",
+      "preview": true
+    },
+    "parameters": {
+      "effect": {
+        "type": "String",
+        "metadata": {
+          "displayName": "Effect",
+          "description": "Enable or disable the execution of the policy"
+        },
+        "allowedValues": [
+          "enforceOPAConstraint",
+          "disabled"
+        ],
+        "defaultValue": "enforceOPAConstraint"
+      },
+      "excludedNamespaces": {
+        "type": "Array",
+        "metadata": {
+          "displayName": "Namespace exclusions",
+          "description": "List of Kubernetes namespaces to exclude from policy evaluation. Providing a value for this parameter is optional."
+        },
+        "defaultValue": []
+      },
+      "allowedVolumeTypes": {
+        "type": "Array",
+        "metadata": {
+          "displayName": "Allowed Volume Types",
+          "description": "List of volume types that volumes are allowed to use. Use * to allow all volume types"
```

Comment:
Does empty input array mean none of the volumes allowed? If yes...
1. Can you check this case through portal? I think portal doesn't allow empty array if empty array is not specified as defaultValue.
2. Can you also add to parameter description that empty array means all volumes are disallowed.